src/components/App.tsx

```diff
@@ -1281,25 +1281,26 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     },
     { replaceAll = false }: { replaceAll?: boolean } = {},
   ) => {
+    const { elements: remoteElements, appState } = sceneData;
```

Comment:
trouble with this naming is that the elements aren't necessarily `remote` so it'll confuse us in the future.

Reply:
I guess depends on what we understand as `remote`.. if anything not coming from within the component, then yeah.

Reply:
But then we'd have to name `remoteElements` even those coming from `initialData` etc. Personally I'd keep `remoteElements` naming only for collaboration/server.

Reply:
with remote I meant anything coming from collab update that's how it was named earlier( in master) as well I think and that's how `handleRemoteSceneUpdate` is named so I changed it to `remoteElements` as it was earlier. Does that make sense?

Reply:
But if it's getting confused as now it will be coming from the host app which so I can change it to `sceneElements` ?

Reply:
Yea, now it's used both by collab (us) and host apps so it can be anything. I'd not destructure it at all, and keep referencing the `sceneData.elements` so it's obvious where it's coming from. You can rename the `sceneData` to `updateData` if you want.

Reply:
Reverted the changes. I generally prefer destructuring at the top so it's clear what all variables would be used below to the reader.

Reply:
Yea, although you then need to come up with names which may/may not be semantically correct, plus when reading the code it may not be obvious where a given destructured variable is coming from. It comes down to preference I guess.

Anyway let's mere this. Thanks :heart:

Reply:
yep, mostly it does boil down to preferences apart from cases where destructuring can impact perf when variables are referenced repeatedly (ex inside a loop).